README.adoc

```diff
@@ -93,25 +96,43 @@ application. Did you notice that there is not a single line of XML? There is no
 file, either. This web application is 100% pure Java and you did not have to deal with
 configuring any plumbing or infrastructure. Spring Boot handles all of that for you.
 
-Logging output is displayed. The service should be up and running within a few seconds.
+=== Getting up and going with wrappers
+The easiest way to get started is to use the *Maven or Gradle wrapper*. It will download 
+all the necessary dependencies and start the Spring Boot app (by running the `main()` 
+method) for you.
+
+==== Running using the Maven Wrapper
+Start the application by running `./mvnw spring-boot:run` on the command line in the root of the project. 
+
+==== Running using the Gradle Wrapper
+Start the application by running `./gradlew bootRun` on the command line in the root of the project.
+
+
+Once you start the application, logging output will be displayed and the service should be up 
+and running within a few seconds. You'll know it's ready once you see a line that reads 
+something like `Started TestingRestdocsApplication in 1.492 seconds`.
+
 
 == Test the Application
 
 Now that the application is running, you can test it. You can load the home page at
 `http://localhost:8080`. However, to give yourself more confidence that the application
 works when you make changes, you want to automate the testing. You also want to publish
-documentation for the HTTP endpoint. You can generate the dynamic parts of that testing as
+documentation for the HTTP endpoint. You can generate the dynamic parts of that documentation as
 part of the tests by using Spring REST Docs.
 
 The first thing you can do is write a simple sanity check test that fails if the
-application context cannot start. To do so, add Spring Test and Spring REST Docs as
-dependencies to your project, in the test scope. The following listing shows what to add
-if you use Maven:
+application context cannot start.
+
+=== Maven
+
+If you use Maven, the following listing shows what to add to the `<dependencies>` 
+section of `pom.xml`:
 
 ====
 [source,xml]
 ----
-include::complete/pom.xml[tag=test]
+include::complete/pom.xml[tag=restdocs-dependency]
 ----
 ====
 
```

Comment:
This section about adding dependencies to Maven/Gradle would go away, because Initializr would've already added them—unless I'm mistaken.

Let us not forget to 
- [ ] remove the asciidoctor tags from the complete version once it's all ready.